Library/Services/Implementation/NfieldDomainLanguageTranslationsService.cs

```diff
@@ -0,0 +1,102 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using Newtonsoft.Json;
+using Nfield.Extensions;
+using Nfield.Infrastructure;
+using Nfield.Models;
+using Nfield.SDK.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nfield.Services.Implementation
+{
+    /// Using LanguageTranslations to keep the same name used for surveys
+    class NfieldDomainLanguageTranslationsService : INfieldDomainLanguageTranslationsService, INfieldConnectionClientObject
+    {
+        private INfieldHttpClient Client
+        {
+            get { return ConnectionClient.Client; }
+        }
+
+        /// <summary>
+        /// Implementation of <see cref="INfieldDomainLanguageTranslationsService"/>
+        /// </summary>
+
+        #region Implementation of INfieldConnectionClientObject
+
+        public INfieldConnectionClient ConnectionClient { get; internal set; }
+
+        public void InitializeNfieldConnection(INfieldConnectionClient connection)
+        {
+            ConnectionClient = connection;
+        }
+
+        #endregion
+
+        #region Implementation of INfieldLanguageTranslationsService
+
+        public Task<IQueryable<Language>> QueryAsync()
+        {
+            return Client.GetAsync(LanguageTranslationsApi())
```

Comment:
Funny, do we as of today still have to deal with task continuation instead of async/await because of clients that depend on a sufficiently old version of .NET?

Reply:
To be honest, I don't know. :-/  We took implementation of other services as an example and ran along with that (so, it's just stupid copypasta). It might be nice to throw a little investigation about .NET versions in use by our API consumers.